JobConfig/primary/flateminus.fcl

```diff
@@ -1,8 +1,21 @@
-# flat range of eminus near the DIO endpoint
-#include "Production/JobConfig/primary/flatE.fcl"
-process_name: flateminus
-physics.producers.generate.physics.genId : dioTail
-physics.producers.generate.physics.pdgId : 11
-physics.producers.generate.physics.elow : 75.0 // Mev
-physics.producers.generate.physics.ehi : 110.0 // Mev
+#
+# generate and produce Detector Steps from flat spectrum using mu- target stops
+#
+# original author: S Middleton
+#
+#include "Production/JobConfig/primary/TargetStopParticle.fcl"
+physics.producers.generate : {
+  module_type : FlatMuonDaughterGenerator
+  inputSimParticles: TargetStopResampler
+  stoppingTargetMaterial : "Al"
+  verbosity : 0
+  makeHistograms : true
+  processcode : "mu2eMuonDecayAtRest"
+  pdgId : -11
+}
+
+physics.producers.generate.startMom : 0
+physics.producers.generate.endMom : 100
+physics.filters.TargetStopResampler.mu2e.MaxEventsToSkip : 1863000  # based on MDC2020, file of 2e8 Beam Resamples
```

Comment:
Specifying the skip in core fcl is deprecated, it should be set based on the dataset, see Production/MDC2020/gen_CeEndpoint.sh for an example.  Eventually this functionality will be included in a POMS campaign.

Reply:
So you want me to just delete this line?

Reply:
I will show you how to replace it within a POMS campaign later.